src/Http/Controllers/SeatNotificationsController.php

```diff
@@ -56,7 +56,7 @@ public function index()
     public function postSubscribe(SubscribeRequest $request, SubscribeAction $action)
     {
 
-        if(empty($request->input('driver_id'))) {
+        if (! empty($request->input('group_id')) && empty($request->input('driver_id'))) {
```

Comment:
curious why do you need `empty($request->input('group_id'))` ? I was assuming that the post for a public channel always have a `driver_id`?

Reply:
Since the `driver_id` is the place where you have to send the notification, I suppose it is a mandatory field?

Reply:
You are right that a `driver_id` is required to subscribe. 
However this if statement takes care of the case if the `driver_id` is missing. That can only be the case for users which have never before oAuthed against the provider. 

* Public subscription should always have a driver_id present (taken from the channel list)